### PR TITLE
feat(integrations): add support for Unipile 

### PR DESCRIPTION
**Pull request by gkhngyk**
What is Unipile?

[Unipile](https://unipile.com) is an innovative API designed to provide seamless integrations of various messaging and communication channels into your software applications.

With Unipile, you can effortlessly incorporate messaging services from popular platforms such as WhatsApp, LinkedIn, Instagram, Messenger, Twitter, and Telegram.

And Unipile also provide comprehensive email integration with all providers, including Google, Microsoft, and IMAP.

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -5929,6 +5929,38 @@ unauthenticated:
...
     docs: https://docs.nango.dev/integrations/all/unauthenticated
 
+unipile:
+    display_name: Unipile
+    categories:
+        - crm
+        - marketing
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://${connectionConfig.subdomain}.unipile.com:${connectionConfig.port}
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:6127]: the port is weird, we need to check if we support that pattern

File: packages/webapp/public/images/template-logos/unipile.svg
Comment (bodinsamuel) [inline packages/webapp/public/images/template-logos/unipile.svg:1]: the logo needs to be transparent, seems like they have an svg ready 
https://www.unipile.com/wp-content/uploads/2023/07/Logo.svg

File: docs-v2/integrations/all/unipile.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+API configuration: [`unipile`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | 🚫 |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | 🚫 |
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/unipile.mdx]: can you change that to be the same format of all the others (all checked except webhooks)

**Review by bodinsamuel — COMMENTED** [review]
Thanks for your contribution, almost good 🙏🏻 
It's missing the category and mint.json update, you check previous similar PR https://github.com/NangoHQ/nango/pull/2962/files

File: packages/webapp/public/images/template-logos/unipile.svg
Comment (gkhngyk) [reply packages/webapp/public/images/template-logos/unipile.svg:1]: I uploaded new logo. Thank you

File: docs-v2/integrations/all/unipile.mdx (hunk 2 above)
Comment (gkhngyk) [reply docs-v2/integrations/all/unipile.mdx]: I updated the list

**Comment by gkhngyk** [comment]
@bodinsamuel Sorry for the late response. I am excited about my first contribution to this wonderful community. I have made the relevant changes and submitted the commit.

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**